src/mixer/playermanager.cpp

```diff
@@ -580,8 +580,11 @@ void PlayerManager::slotLoadTrackToPlayer(TrackPointer pTrack, QString group, bo
     }
 
     mixxx::Duration elapsed = m_cloneTimer.restart();
-    if (m_lastLoadedPlayer == group && elapsed < mixxx::Duration::fromSeconds(0.5)) {
-        // load pressed twice quickly, clone instead of loading
+    bool cloneOnDoubleTap = m_pConfig->getValue<bool>(
```

Comment:
I think we should here use the defaut value if the config is missing. There should be a function that allows to set a default as fall back.

Reply:
using the default (off) now.
but shouldn't defaults be **set** only in dlgprefdeck.cpp?